### PR TITLE
Auto focus when user needs to interact with the app

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch8583]

File: src/crypto/accounts.slice.test.ts [hunk 1]
```diff
@@ -17,6 +16,7 @@ import {
   setGeneratedAccount
 } from '@common/store';
 import { DEFAULT_MNEMONIC_INDEX } from '@config';
+import { createWallet, derivePrivateKey, getAddress, getAddresses } from '@crypto/crypto';
```
Comment (Mrtenz) [inline src/crypto/accounts.slice.test.ts:19]: You can use a relative import here (and in some other files as well).

File: src/window.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+export const showWindowOnTop = (window: BrowserWindow) => {
+  window.setAlwaysOnTop(true);
+  // Hack to show on current desktop - see: https://github.com/electron/electron/issues/5362
+  window.setVisibleOnAllWorkspaces(true, { visibleOnFullScreen: true });
+  window.show();
+  window.focus();
+  window.setVisibleOnAllWorkspaces(false);
+  window.setAlwaysOnTop(false);
+};
```
Comment (Mrtenz) [inline src/window.ts:11]: What's the reason for this being a separate file?

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: src/window.ts (hunk 2 above)
Comment (FrederikBolding) [reply src/window.ts:11]: Just for reusability. It is used in `main.ts` and the saga.

Comment (Mrtenz) [reply src/window.ts:11]: Can you move it to `utils` then?

Comment (FrederikBolding) [reply src/window.ts:11]: Sure

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**